src/pmpo_c.cpp

```diff
@@ -607,13 +601,13 @@ void polympo_setMeshVel_f(MPMesh_ptr p_mpmesh, const int nComps, const int nVert
   auto coordsArray = p_mesh->getMeshField<polyMPO::MeshF_Vel>();
   auto h_coordsArray = Kokkos::create_mirror_view(coordsArray);
   for(int i=0; i<nVertices; i++){
-    h_coordsArray(i,0) = array[i*vec2d_nEntries];
-    h_coordsArray(i,1) = array[i*vec2d_nEntries+1];
+    h_coordsArray(i,0) = uVelIn[i];
+    h_coordsArray(i,1) = vVelIn[i];
   }
   Kokkos::deep_copy(coordsArray, h_coordsArray);
 }
 
-void polympo_getMeshVel_f(MPMesh_ptr p_mpmesh, const int nComps, const int nVertices, double* array){
+void polympo_getMeshVel_f(MPMesh_ptr p_mpmesh, const int nComps, const int nVertices, double* uVelOut, double* vVelOut){
```

Comment:
we do not need 'nComps'

Reply:
removed https://github.com/SCOREC/polyMPO/commit/8bec689310aa9db8a2ecd8673a002858a0506d28